tests/controls/webtest.cpp

```diff
@@ -302,6 +306,12 @@ void WebTestCase::SetPage()
     CPPUNIT_ASSERT_EQUAL("other text", m_browser->GetPageText());
 }
 
+void WebTestCase::EmulationLevel()
+{
+    CPPUNIT_ASSERT(m_browser->SetEmulationLevel(wxWEBVIEWIE_EMULATION_LEVEL_7_DEFAULT));
+    CPPUNIT_ASSERT_EQUAL(wxWEBVIEWIE_EMULATION_LEVEL_7_DEFAULT, m_browser->GetEmulationLevel());
```

Comment:
You should probably unset the registry entry after this to put it back into the state it was before.